crates/adapters/src/test/mod.rs

```diff
@@ -39,6 +39,16 @@ mod mock_output_consumer;
 
 mod datagen;
 
+#[cfg(all(
+    feature = "with-iceberg",
```

Comment:
do we ever compile without this feature?

Reply:
yes, when testing the SQL compiler